README.md

```diff
@@ -68,6 +68,7 @@ There are many available implementations to choose from, here are some of the mo
     * [`stderrlog`](https://docs.rs/stderrlog/*/stderrlog/)
     * [`flexi_logger`](https://docs.rs/flexi_logger/*/flexi_logger/)
     * [`call_logger`](https://docs.rs/call_logger/*/call_logger/)
+    * [`std-logger`](https://docs.rs/std-logger/*/std-logger/)
```

Comment:
```suggestion
    * [`std-logger`](https://docs.rs/std-logger/*/std_logger/)
```

Reply:
It looks like `docs.rs` interprets the `-` in `std-logger` after the version pattern incorrectly and gives you an error page. This link seems to work correctly.